README.md

```diff
@@ -46,14 +46,14 @@ A `tests.toml` file has this format:
 ```toml
 # This is an auto-generated file.
 #
-# Regenerating this file will:
-# - Update the `description` property
-# - Update the `reimplements` property
-# - Remove `include = true` properties
-# - Preserve any other properties
+# Regenerating this file via `configlet sync` will:
+# - Recreate every `description` key/value pair
+# - Recreate every `reimplements` key/value pair, where they exist in problem-specifications
```

Comment:
I don't love "recreate", since it doesn't communicate very well that the latest value of the description is used.

The problem with just "update" was that the `description` and `reimplements` properties are added even if they didn't previously exist in the `tests.toml` for some reason.